operate/client/src/modules/hooks/processDefinitions.ts

```diff
@@ -72,9 +89,53 @@ function useProcessDefinitionVersions(
   });
 }
 
+function useProcessDefinitionSelection() {
```

Comment:
FIY: This hook and its usage in _ListView/v2/DiagramPanel_ are identical/inspired by `useDecisionDefinitionSelection()`.